frontend/src/component/Sidebar/SidebarSpaceItem.jsx

```diff
@@ -161,45 +163,40 @@ class SidebarSpaceItem extends React.Component {
             customClass={`transparentButton sidebar__item__foldChildren ${LOCK_TOGGLE_SIDEBAR_WHEN_OPENED_ON_MOBILE}`}
             icon={`fas fa-caret-${props.foldChildren ? 'right' : 'down'}`}
             title={props.foldChildren ? props.t('Show subspaces') : props.t('Hide subspaces')}
-            intent='link'
-            mode='light'
-            onClick={() => props.onToggleFoldChildren(props.spaceId)}
+            onClick={(e) => {
+              e.preventDefault()
+              props.onToggleFoldChildren(props.spaceId)
+            }}
```

Comment:
as seen together, e.preventDefault() doesn't seem to be useful (to confirm) and it is lacking a e.stopPropagation() to avoid also executing the onClick={this.handleClickSpace} from the <Link> parent

Reply:
Fixed